tasks/setup/overview.md

```diff
@@ -0,0 +1,35 @@
+# Setup Infracost Task
+
+This Azure Devops Task downloads and installs the Infracost CLI in your Azure Devops Pipeline. Subsequent steps in the same job can run the CLI in the same way it is run on the command line.
+
+## Usage
+
+The task can be used as follows. You probably want to run Infracost CLI commands then use the [comment](TODO link to comment webpage) task to post comments! See the [examples](https://github.com/infracost/infracost-azure-devops/examples) section on github for examples of how these tasks can be combined.
```

Comment:
```suggestion
The task can be used as follows. You probably want to run Infracost CLI commands then use the [comment](TODO link to comment webpage) task to post comments! See the [examples](https://github.com/infracost/infracost-azure-devops/examples) section on GitHub for examples of how these tasks can be combined.
```

Reply:
done in https://github.com/infracost/infracost-azure-devops/pull/9/commits/95030bf9394c362cb56a6bdec90109b014fcd819